reservation/tests.py

```diff
@@ -17,148 +15,131 @@
     typeOfJob_object.save()
-    return [(typeOfJob.typeOfJob_name, typeOfJob.price), (typeOfJob_object.typeOfJob_name, typeOfJob_object.price)]
-
-
-@pytest.fixture
-def appointment(typeOfJob):
-    return Appointment(professional_id=PROFESSIONAL_ID, client_id=CLIENT_ID, typeOfJob_id=typeOfJob,
-                       start_appointment=START_APPOINTMENT, end_appointment=END_APPOINTMENT, summary=SUMMARY)
-
-
-@pytest.fixture
-def persisted_appointment(appointment):
-    appointment.typeOfJob_id.save()
-    appointment.save()
-    return appointment
+    return [(persisted_typeOfJob.typeOfJob_name, persisted_typeOfJob.price),
+            (typeOfJob_object.typeOfJob_name, typeOfJob_object.price)]
 
 
 @pytest.fixture
-def persisted_appointment_pool(persisted_appointment):
-    typeOfJob_object = TypeOfJob(professional_id=PROFESSIONAL_ID, typeOfJob_name="Hair cut", price=100)
-    appointment_object = Appointment(professional_id=PROFESSIONAL_ID,
-                                     client_id=10,
-                                     typeOfJob_id=typeOfJob_object,
+def persisted_appointment_pool(persisted_appointment, typeOfJob, professional, client2):
+    appointment_object = Appointment(professional_id=professional,
+                                     client_id=client2,
+                                     typeOfJob_id=typeOfJob,
                                      start_appointment=datetime(2023, 4, 20, 12, 0, 0),
                                      end_appointment=datetime(2023, 4, 20, 13, 0, 0),
                                      summary=SUMMARY)
-    typeOfJob_object.save()
+    appointment_object.professional_id.save()
+    appointment_object.client_id.save()
+    appointment_object.typeOfJob_id.save()
     appointment_object.save()
-    return [persisted_appointment.client_id, appointment_object.client_id]
+    return [persisted_appointment.client_id.client_id, appointment_object.client_id.client_id]
 
 
 @pytest.fixture
-def schedule():
-    return Schedule(professional_id=PROFESSIONAL_ID,
-                    start_day=START_DAY,
-                    end_day=END_DAY,
-                    meeting_time=MEETING_TIME)
-
-
-@pytest.fixture
-def persisted_schedule_pool(persisted_appointment, schedule):
-    typeOfJob_object = TypeOfJob(professional_id=PROFESSIONAL_ID, typeOfJob_name="Hair cut", price=100)
-    appointment_object1 = Appointment(professional_id=PROFESSIONAL_ID,
-                                      client_id=10,
-                                      typeOfJob_id=typeOfJob_object,
-                                      start_appointment=datetime(2023, 4, 17, 13, 0, 0),
-                                      end_appointment=datetime(2023, 4, 17, 14, 0, 0),
-                                      summary=SUMMARY)
+def persisted_schedule_pool(persisted_appointment, typeOfJob, persisted_schedule, professional, client2):
+    appointment_object = Appointment(professional_id=professional,
```

Comment:
better to call it appointment 

class TestTypeOfJobModel: test_typeof_job_model.py
class TestAppointmentModel: test_appointment_model.py

And so on

Reply:
Done